lms/djangoapps/support/tests/test_views.py

```diff
@@ -1189,3 +1189,177 @@ def test_fbe_disabled_response(self):
         )
         data = json.loads(response.content.decode('utf-8'))
         assert data == {}
+
+
+@ddt.ddt
+class LinkProgramEnrollmentSupportAPIViewTests(SupportViewTestCase):
+    """
+    Tests for the link_program_enrollments support view.
+    """
+
+    def setUp(self):
+        """Make the user support staff. """
+        super().setUp()
+        self.url = reverse("support:link_program_enrollments_details")
+        SupportStaffRole().add_users(self.user)
+        self.program_uuid = str(uuid4())
+        self.username_pair_text = '0001,user-0001\n0002,user-02'
+
+    def _setup_user_from_username(self, username):
+        """
+        Setup a user from the passed in username.
+        If username passed in is falsy, return None
+        """
+        created_user = None
+        if username:
+            created_user = UserFactory(username=username, password=self.PASSWORD)
```

Comment:
username is supplied always, so there is no need for if condition.

Reply:
A value of None or '' can also be passed in the function. Check test: `test_linking_program_enrollment_without_username`